AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -235,10 +235,8 @@ extension DefaultVideoClientController: VideoClientDelegate {
         guard let messages = messages else { return }
         for message in messages {
             let dataMessage = DataMessage(message: message)
-            if let observers = dataMessageObservers[dataMessage.topic] {
-                ObserverUtils.forEach(observers: observers) { (observer: DataMessageObserver) in
-                    observer.dataMessageDidReceived(dataMessage: dataMessage)
-                }
+            if let observer = dataMessageObservers[dataMessage.topic] as? DataMessageObserver {
```

Comment:
This will not work, `dataMessageObservers[dataMessage.topic]` is `NSMutableSet`, not `DataMessageObserver `.

Reply:
Updated. However when I changed it to

```
if let observer = observer as DataMessageObserver {
    observer.dataMessageDidReceived(dataMessage: dataMessage)
}
```

I get `Conditional cast from 'any DataMessageObserver' to 'any DataMessageObserver' always succeeds`

So it's an unnecessary check, but the same check is made within the `forEach` function I added.